src/crimson/os/seastore/journal/segment_allocator.cc

```diff
@@ -207,8 +207,8 @@ SegmentAllocator::close_segment()
   assert(can_write());
   // Note: make sure no one can access the current segment once closing
   auto seg_to_close = std::move(current_segment);
+  DEBUG(" {}", seg_to_close->get_segment_id());
```

Comment:
This is a duplicated log, see line 237 about the INFO log below.

Reply:
Yes, removed.